content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+
+.. image:: google_oauth/app-name-odoo.png
+   :class: img-fluid
+   :align: center
+   :alt: App Name and User Support Email on Google OAuth.
+
+- Next, under :menuselection:`App Domain --> Authorized domains`, click on :guilabel:`Add Domain` and
+  enter *odoo.com*.
+- After that, under the :guilabel:`Developer contact information` section, enter the organization's
+  email address. Google uses this email address to notify the organization about any changes to
+  your project.
+- Next, click on the :guilabel:`Save and Continue` button.
+- Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
+  :guilabel:`Save and Continue`.
+- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
+  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
+  :guilabel:`App registration` appears.
+- Finally, scroll to the bottom and click on :guilabel:`Back to Dashboard`.
+
+Create Credentials
+------------------
+
+Now that you have set up the project, you need to create credentials, which includes the
+:guilabel:`Client ID` and :guilabel:`Client Secret`. First, click on :guilabel:`Credentials` in the
+left sidebar menu.
+
+Then, click on :guilabel:`Create Credentials` in the top menu and select
+:guilabel:`OAuth client ID` from the dropdown menu.
+
+.. image:: google_oauth/create-credentials.png
+   :class: img-fluid
+   :align: center
+   :alt: Create Credentials on Google OAuth.
+
+- Under :guilabel:`Application Type`, select :guilabel:`Web Application` from the dropdown menu.
+- In the :guilabel:`Client ID` field, enter *Odoo*.
+- Under the :guilabel:`Authorized redirect URIs` label, click the button :guilabel:`ADD URI`, and
+  then input *https://yourdbname.odoo.com/google_gmail/confirm* in the :guilabel:`URIs 1` field. Be
+  sure to replace the *yourdbname* part of the URL with your actual Odoo database name.
+
+.. image:: google_oauth/add-uri.png
+   :class: img-fluid
+   :align: center
+   :alt: Add URI in Creating Credentials for Google OAuth.
+
+- Next, click on :guilabel:`Create` and Google will generate an OAuth :guilabel:`Client ID` and
+  :guilabel:`Client Secret`. Finally, copy each generated value to a Notepad and navigate to the
+  Odoo database.
+
+.. image:: google_oauth/client-credentials.png
+   :class: img-fluid
+   :align: center
+   :alt: Client ID and Client Secret for Google OAuth.
+
+Setup in Odoo
+=============
+
+Enter Google Credentials
+------------------------
+
+First, open Odoo and navigate to the :guilabel:`Apps` module. Then, remove the
+:guilabel:`Apps filter` and type in *Google*. Install the module called :guilabel:`Google Gmail`.
+
+Next, navigate to the :menuselection:`Settings module --> General Settings`, and under the
+:guilabel:`Discuss` section, ensure that the checkbox for :guilabel:`Custom Email Servers` is
+checked. This populates a new option for :guilabel:`Gmail Credentials`. Here, copy and paste the
```

Comment:
* after `:guilabel:'Gmail Credentials'.`, add a sentence `Make sure to check the box next to it.` 
* replace `Here` with `Then`